pkgs/tools/misc/bat/default.nix

```diff
@@ -15,11 +15,15 @@ rustPlatform.buildRustPackage rec {
 
   nativeBuildInputs = [ cmake pkgconfig zlib ];
 
+  buildInputs = [] ++ stdenv.lib.optionals stdenv.isDarwin [
+    libiconv darwin.apple_sdk.frameworks.Security
```

Comment:
Can you put libiconv in buildInputs? The conditional should not be necessary.

Reply:
Sure, will do.
FWIW: this was inspired by [`exa`](https://github.com/NixOS/nixpkgs/blob/c958516cb3e8b68433097c15f49eedcac781b93d/pkgs/tools/misc/exa/default.nix#L23) which also only includes `libiconv` conditionally.